spec/hash-map.dd

```diff
@@ -361,9 +361,13 @@ $(SPEC_RUNNABLE_EXAMPLE_RUN
 
 $(H2 $(LNAME2 static_initialization, Static Initialization of AAs))
 
-        NOTE: Not yet implemented.
+        NOTE: Static initialization, e.g. assigning a global variable with some
```

Comment:
```suggestion
        NOTE: Static initialization, i.e. assigning a global variable with some
```
I think "assigning a global variable" is the only example of static initialization, or are there others?

Reply:
`switch (foo) { case ["foo": "bar"]: x; }` would be one I think, but I don't think switches are even usable with AAs.

`static` variables would be another example, other than global variables. (even if they are basically the same)

Reply:
improved the wording, included `static` as example.